src/data/store.ts

```
@@ -156,6 +167,45 @@ export function data(
         });
       }
 
+      // If this action wants us to update certain queries. Let’s do it!
+      if (constAction.updateQueries) {
+        Object.keys(constAction.updateQueries).forEach(queryId => {
+          const query = queries[queryId];
+          if (!query) {
+            // XXX should throw an error?
```

I think the only case in which this could happen is if the query was removed from queries between the time we look up the queryId from queryName and the time we reach this code. Since actions are dispatched synchronously that should never happen, right?
It's prudent to keep this check here, but I don't think we should throw an error.